basketball_reference_web_scraper/parsers/box_scores/play_by_play.py

```diff
@@ -0,0 +1,38 @@
+import re
+from lxml import html
+
+from basketball_reference_web_scraper.data import Location
+
+
+TIME_REGEX = "([0-9]+):([0-9]+)\.([0-9]+)"
+SCORE_REGEX = "([0-9]+)-([0-9]+)"
+
+
+def parse_play_by_plays(page):
```

Comment:
![image](https://user-images.githubusercontent.com/8136030/60858980-88f68f80-a1c5-11e9-99fa-b6a0196bd392.png)

FYI, jump balls don't seem to be logged.

Totally fine - definitely something we / I to add later.

Reply:
I ignored it for now because if we want to support this, we need to implement a third Location - BOTH, as it is the only event that occur for both sides